README.md

```diff
@@ -4,39 +4,53 @@
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/devconsole-operator/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/devconsole-operator)
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/operator-registry/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/operator-registry)
 
-DevConsole operator enables a developer-focused view in OpenShift 4.
-It provides a view switcher to transition between the traditional
-Kubernetes cluster administration console referred to as
-Administrator, to this new Developer perspective.
+## Overview
+The DevConsole operator enables a developer-focused view in the OpenShift 4 web console.
+It provides a view switcher to transition between the traditional Kubernetes cluster administration console referred to as Administrator, to a new Developer perspective.
```

Comment:
Longer lines are harder to review and comment. Also not easy to understand the difference. Please wrap the lines.

Reply:
If you had kept the line breaks. It looks likes the second and third line was not changed.